mcp-server/src/core/utils/path-utils.js

```diff
@@ -89,7 +89,9 @@ export function findTasksJsonPath(args, log) {
 
 	// 1. If project root is explicitly provided (e.g., from MCP session), use it directly
 	if (args.projectRoot) {
-		const projectRoot = args.projectRoot;
+		// Decode URL-encoded characters (like %20 to spaces) in projectRoot
+		// Fix for issues #192 and #246
+		const projectRoot = decodeURIComponent(args.projectRoot);
```

Comment:
Did you make sure that it also works inside windows ? For example, I saw people had an issue where the path was something like /c:/... Does that work with it as well ?